pkg/catalogsourceconfig/configuring.go

```diff
@@ -207,9 +207,8 @@ func newConfigMap(csc *v1alpha1.CatalogSourceConfig, data map[string]string) *co
 func newCatalogSource(csc *v1alpha1.CatalogSourceConfig, configMapName string) *olm.CatalogSource {
 	builder := new(CatalogSourceBuilder).
 		WithOwner(csc).
-		WithMeta(v1alpha1.CatalogSourcePrefix+csc.Name, csc.Spec.TargetNamespace).
-		// TBD: where do we get display name and publisher from?
```

Comment:
Maybe this comment should remain? I know we are hardcoding them for now, but the problem still exists if we want to make it come from somewhere dynamic (real datastore?)

Reply:
Done.